modules/standard/SysBasic.chpl

```diff
@@ -33,64 +59,115 @@ extern type socklen_t = int(32);
 
 // stdin/stdout/sterr
 //extern proc chpl_cstdin():_file;
+pragma "no doc"
 extern proc chpl_cstdout():_file;
+pragma "no doc"
 extern proc chpl_cstderr():_file;
+pragma "no doc"
 extern proc chpl_cnullfile():_file;
 
-// system error number.
+// chpldoc TODO - docs for syserr don't show up
+/* A type storing an error code or an error message.
+   A syserr can be compared using == or != to an err_t (ie integer error code)
+   or to another syserr. A syserr can be cast to or from an err_t. It can be
+   assigned the value of an err_t or another syserr. In addition, syserr can be
+   checked directly in an if statement like so:
```

Comment:
huh? Are you saying it should read "syserr can be checked directly into an if statement"? That doesn't make sense to me...